lib/probes/http.js

```diff
@@ -566,7 +566,7 @@ function patchServer (module, options, protocol) {
       if (span.doMetrics) {
 
         // set this value only if doing metrics.
-        exitKeyValuePairs.TransactionName = res._ao_metrics.txName
+        exitKeyValuePairs.TransactionName = res._ao_metrics.txname;
```

Comment:
Do we need to make a regular (non-lambda) agent release to get this fix in?

Reply:
yes. that release will be based on this lambda code base.